addons/circulation/functions/fnc_checkBloodPressureLocal.sqf

```diff
@@ -0,0 +1,57 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Glowbal modified by Cplhardcore
+ * Local callback for checking the blood pressure of a patient.
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ * 2: Body Part <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, cursorObject, "LeftArm"] call kat_circulation_fnc_checkBloodPressureLocal
+ *
+ * Public: No
+ */
+
+params ["_medic", "_patient", "_bodyPart"];
+
+private _bloodPressure = [0, 0];
+
+if (alive _patient && {!([_patient, _bodyPart] call ACEFUNC(medical_treatment,hasTourniquetAppliedTo))}) then {
+    _bloodPressure = [_patient] call FUNC(getBloodPressure);
+};
+
+private _bloodPressureOutput = ACELSTRING(medical_treatment,Check_Bloodpressure_Output_6);
+private _logOutput = ACELSTRING(medical_treatment,Check_Bloodpressure_NoBloodpressure);
+
+_bloodPressure params ["_bloodPressureLow", "_bloodPressureHigh"];
+
+if (_bloodPressureLow > 40) then {
+    if (_medic call ACEFUNC(medical_treatment,isMedic)) then {
+        _bloodPressureOutput = LSTRING(Check_Bloodpressure_Output_Palp);
+        _logOutput = format [localize "STR_KAT_Circulation_Bloodpressure_Output_Palp", (round (_bloodPressureLow / 10) * 10) + (if (random 1 > 0.5) then {10} else {-10})];
```

Comment:
first, why did you not use LSTRING?
secondly, whats the point of (_bloodPressureLow / 10) * 10)?
and last, (if (random 1 > 0.5) then {10} else {-10}) can be converted to use select